src/sentry/relay/config/metric_extraction.py

```diff
@@ -150,7 +151,7 @@ def _get_widget_metric_specs(
 ) -> List[HashedMetricSpec]:
     if not (
         "organizations:on-demand-metrics-extraction" in enabled_features
-        and "organizations:on-demand-metrics-extraction-experimental" in enabled_features
+        and "organizations:on-demand-metrics-widgets-extraction" in enabled_features
```

Comment:
I wonder if we should completely decouple from `organizations:on-demand-metrics-extraction`.